### DISCUSSION TITLE
Lesson 6 foundry with ganache

### DISCUSSION THREAD

**Discussion by fethallaheth** [Cyfrin/foundry-full-course-cu#136] [Q&A]
i connect foundry with ganache

Error
'''$ forge create SimpleStorage --rpc-url http://127.0.0.1:7545 --interactive
[⠑] Compiling...
No files changed, compilation skipped
Error:
The handle is invalid. (os error 6)'''

**Comment by ShayanShamsi** [comment]
Hmm... can not give a concrete conclusion yet. Which operating system are you using?

**Reply by fethallaheth** [reply]
windows 
bash

**Reply by ShayanShamsi** [reply]
I did some Google-ing and could not find a definite answer. Everyone seems to have a very specific fix for their situation.
Stackoverflow user [nkamado](https://stackoverflow.com/users/569849/nkamado) suggested in his answer on [this thread](https://stackoverflow.com/questions/3028786/how-can-i-fix-error-6-the-handle-is-invalid-with-pyserial#:~:text=This%20problem%20is%20caused%20by,modules.) that:
`
This problem is caused by the invalid handle casting which discard the upper 32-bit of 64-bit value...
`

Maybe you should create an issue on the Foundry repo?

**Reply by fethallaheth** [reply]
yeah ill do thanks for ur help

**Reply by ShayanShamsi** [reply]
I hope you figure it out or someone can help you. If not, then let me know and we can try our best!

**Reply by fethallaheth** [reply]
i create an issue on foundry official repo ;
ill do thanks again

**Comment by Equious** [comment]
Hey @fethallaheth. Did you find a resolution here? Let us know if we can help any further 🙏
